packages/client/src/disposable.ts

```diff
@@ -3,31 +3,29 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
 
-import { Disposable } from "vscode-jsonrpc";
-
-export {
-    Disposable
-}
+import { Disposable } from 'vscode-jsonrpc';
 
 export class DisposableCollection implements Disposable {
     protected readonly disposables: Disposable[] = [];
 
-    dispose(): void {
+    dispose (): void {
         while (this.disposables.length !== 0) {
-            this.disposables.pop()!.dispose();
+            this.disposables.pop()?.dispose();
```

Comment:
What was the issue with the `!`?

Reply:
None, my error, happend after first round of lint issue resolution. I will change back.